README.md

```diff
@@ -20,37 +34,32 @@ at the conclusion of the tests by default.
 
 Apply the resource claim as follows to re-create
 the namespace, Prometheus, Grafana and dependencies for further
-exploration.
+exploration. Note that the xmetrics configuration examples
+rely on the CRDs to be installed through the oss composition
+first.
 ```
 kubectl apply -f examples/oss.yaml
```

Comment:
Is this claim installing x-metrics? If so, change the name. OSS is very generic, I would avoid using it.

Reply:
Would you recommend to not use oss.yaml for the claim name, even though x-metrics, prometheus and grafana are open source?

If so, which name would resonate with you?